ui/helpers/utils/util.js

```diff
@@ -588,3 +588,14 @@ export function coinTypeToProtocolName(coinType) {
   }
   return slip44[coinType]?.name || undefined;
 }
+
+/**
+ * Tests "nullishness". Used to guard a section of a component from being
+ * rendered based on a value.
+ *
+ * @param {any} value - A value (literally anything).
+ * @returns `true` if the value is null or undefined, `false` otherwise.
+ */
+export function isNullish(value) {
```

Comment:
I added this function because I found that in a lot of places whenever we optionally display pieces of a component we do something like this:

``` javascript
foo && (
  <div>{/* ... */}
)
```

or:

``` javascript
foo ? (
  <div>{/* ... */}
) : null
```

However what if `foo` is 0 or '0' or something to that nature? Instead in my opinion we should be testing for nullishness:

``` javascript
isNullish(foo) ? null : (
  <div>{/* ... */}
) 
```